pkgs/desktops/cinnamon/cinnamon-session/default.nix

```diff
@@ -0,0 +1,51 @@
+{ cinnamon-desktop, cinnamon-settings-daemon, dbus-glib, docbook_xsl, docbook_xml_dtd_412, fetchFromGitHub, glib, gsettings_desktop_schemas, gtk3, libcanberra, libxslt, makeWrapper, meson, ninja, pkgconfig, python3, stdenv, systemd, wrapGAppsHook, xapps, xmlto, xorg, gnome2, cmake, libexecinfo }:
+
+stdenv.mkDerivation rec {
+  pname = "cinnamon-session";
+  version = "4.2.1";
+
+  src = fetchFromGitHub {
+    owner = "linuxmint";
+    repo = pname;
+    rev = version;
+    sha256 = "1jrwjnrcmp5m9vlp42ql79bxic5nrs37kkgcvgyhvvvskvdwpyfw";
+  };
+
+  /* Run-time dependency libelogind found: NO (tried pkgconfig and cmake)
+  Run-time dependency libsystemd-login found: NO (tried pkgconfig and cmake)
+  Run-time dependency libsystemd found: NO (tried pkgconfig and cmake) */
+
+  buildInputs = [ cinnamon-desktop cinnamon-settings-daemon dbus-glib docbook_xsl docbook_xml_dtd_412 glib gsettings_desktop_schemas gtk3 libcanberra libxslt makeWrapper pkgconfig python3 xapps xmlto xorg.xtrans xorg.libXtst gnome2.GConf systemd ];
```

Comment:
it's possible  docbook_xsl docbook_xml_dtd_412 python3 should be native.

Reply:
gsettings_desktop_schemas is an alias to gsettings-desktops-schemas

Reply:
Maybe add `nixpkgs.config.allowAliases = false;` to your `configuration.nix`, or `allowAliases = false;` to your `~/.config/nixpkgs/config.nix`.